windows/RNDeviceInfo/RNDeviceInfoModule.cs

```diff
@@ -111,6 +111,7 @@ private bool IsTablet(string os)
                 constants["timezone"] = TimeZoneInfo.Local.Id;
                 constants["isEmulator"] = IsEmulator(model);
                 constants["isTablet"] = IsTablet(os);
+                constants["isDeveloperModeEnabled"] = false;
```

Comment:
Let's revert and leave this file unchanged, so that isDeveloperModeEnabled() returns `undefined` which is a good behavior when a feature is not implemented